specs/commandline/inspect.md

```diff
@@ -0,0 +1,271 @@
+# notation inspect
+
+## Description
+
+Use `notation inspect` command to inspect all the signatures associated with signed artifact in a human readable format.
```

Comment:
Can `notation inspect` inspect a specific signature?

Reply:
Users may be interested in only one signature after running `notation list`, especially when there are lots of signatures associated with the artifact.

Reply:
In other words, it seems that `notation list` is a proper subset of `notation inspect`. Or, `notation inspect` is more like `notation list --verbose`. /cc @yizha1 @FeynmanZhou

Reply:
+1. The description of `notation inspect` looks quite similar to [notation list](https://github.com/notaryproject/notation/blob/main/specs/commandline/list.md#description). Users might be confused about the differences between these two.

Reply:
> Can `notation inspect` inspect a specific signature?

Will be addressed in the new issue for the upcoming release.

Reply:
https://github.com/notaryproject/notation/issues/506 - New Issue created for the single signature inspect.